streams/src/main/java/org/apache/kafka/streams/kstream/internals/ConsumedInternal.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.kafka.streams.kstream.internals;
 
+import org.apache.kafka.common.serialization.Deserializer;
```

Comment:
This import is not necessary, and is causing jenkins to fail.

Reply:
Removed the unused import.